test/e2e/specs/site-editor/template-revert.spec.js

```diff
@@ -248,7 +248,9 @@ class TemplateRevertUtils {
 						record.blocks
 					);
 				} else if ( record.content ) {
-					return record.content;
+					return window.wp.blocks.__unstableSerializeAndClean(
+						window.wp.blocks.parse( record.content )
+					);
```

Comment:
This necessary change actually shows that the PR is working! Previously, both the edited entities before and after the template change would have the `blocks` property. That's because on mount, the query block was editing attributes, and thus creating an "edit". This is no longer the case after the changes in this PR, so the initial entity has no `blocks` property. Instead, the test will use `content`, but there's some whitespace and attribute order differences. The solution is to parse this content and serialise it the same way as `blocks` a few lines above.